eu.betterbird.Betterbird.json

```
@@ -0,0 +1,72 @@
+{
+    "app-id": "eu.betterbird.Betterbird",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/bin/betterbird",
+    "rename-icon": "betterbird",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
```

```suggestion
        "--share=ipc", "--socket=x11",
        "--socket=wayland",
```

does it support wayland **out of the box** ?

---

Wayland support is still experimental, but can be enabled at runtime: https://github.com/flathub/org.mozilla.Thunderbird/issues/75

---

then it should be remove until it's production ready.

---

Ok, done.

---

Both Firefox and Thunderbird flatpaks have wayland permission even if it's not used by default so I think it shouldn't be different here. Many people enable wayland backend for those apps and it's easier for them if there is no extra steps involved.